src/site/_includes/partials/post.njk

```diff
@@ -11,7 +11,18 @@
   {% endif %}
 
   {# Make sure Actions are first in the tab order. #}
-  <web-actions actions="share|subscribe" authors="{{ authors | expandContributors | join('|') }}"></web-actions>
+  <div class="w-actions">
+    <share-action authors="{{ authors | expandContributors | join('|') }}"
+      class="gc-analytics-event w-actions__fab w-actions__fab--share"
+      data-category="web.dev"
+      data-label="live, share"
```

Comment:
tiny nit: this should have a different label ("live" was for the live page)

Actually, previously we had separate Analytics events for Web Share vs Twitter share. I think we can just have a default and let the WC modify its own property.

I'll push a PR to this branch to add this.